TicTacToe.cabal

```diff
@@ -0,0 +1,64 @@
+cabal-version: 1.12
```

Comment:
Seems like an overkill to use both stack and cabal. It's not as if you want to support both for different packaging / repositories etc.

Reply:
This is just what the default `stack new` does